### DISCUSSION TITLE
[Suggest] Structure the moleculerjs project

### DISCUSSION THREAD

**Discussion by lucwj** [moleculerjs/educational-content#9] [Ideas]
# This is a guide helping you to write an article

> Structuring the moleculerjs project is not quite easy. With the nice project structure we can easy to scale up the team, easy to develop, easy to deploy, ...

Your article should have the following structure:

## Structure

- Title
- Introduction (briefly explain what people will learn in this article)
- Goals (explain what does the person "gain" by reading the article/tutorial)
- Prerequisites (mention the software versions that are required and the necessary knowledge background)
- Step-by-step instructions (explain how to achieve the above mentioned goal)
- Provide a link to the source code of your article/tutorial (e.g., github)
- Conclusion

**Comment by AndreMaz** [comment]
Sorry, currently this topic is not on our priority list and, therefore, it not covered by the community program. However, we would love to hear your testimonials and expertise about this topic.
